ffi/dotnet-idsdk/IoticsIdLib/README.md

```diff
@@ -0,0 +1,29 @@
+# Build & run
+
+1. go to the ffi directory and run the Makefile or the build.ps1 as per README.
+2. once the build of the ffi wrapper works, copy the libraries in this top directory `copy .\lib\*.* .\dotnet-idsdk\IoticsIdLib\`
+    - the libraries need to be made available to the project. there should be a way to configure the path in the csproj but I haven't found a way yet
+3. run
+    - `dotnet clean` to clean the project (sometimes it seems you need to delete the directories `bin` and `obj` manually)
+    - `dotnet build` to build it
+    - `dotnet run` to run the Main method in `src\Main.cs`
+
+In order to distribute the project to 3rd parties you should zip the two files `lib-iotics-id-sdk.[dll|h]` and the content of the bin directory.
+
+
+## Run output
+
+a working output for the run is 
```

Comment:
This shouldn't be here like so. (Surely if any of the commands don't work it'll produce an exception & exit?)